distro/pom.xml

```diff
@@ -49,6 +49,9 @@
     <ordertemplates.version>1.0.2</ordertemplates.version>
     <patientflags.version>3.0.7</patientflags.version>
     <o3forms.version>2.3.0-SNAPSHOT</o3forms.version>
+    <!-- Stock Management and Billing -->
+    <stockmanagement.version>2.0.1-SNAPSHOT</stockmanagement.version>
+    <billing.version>1.1.0</billing.version>
```

Comment:
Has version `1.1.0` of the billing module been released?

Reply:
Yes, this was released

Reply:
I failed to find such a release from this repository: https://github.com/openmrs/openmrs-module-billing
Or are you using another repository?

Reply:
@dkayiwa  - I think there was a manual release on BambooCI, [link](https://ci.openmrs.org/browse/BILL-BILL-13)

@ODORA0  - The task `Build and Validate Configuration` is failing because some privileges (likely) reference Roles that may not be in the Roles iniz domain. Please check

Reply:
> I think there was a manual release on BambooCI, [link](https://ci.openmrs.org/browse/BILL-BILL-13)

Could there be a reason why you set this to `false`? https://github.com/openmrs/openmrs-module-billing/blob/main/pom.xml#L367

For I notice that no tag was created on github and the billing module is still having 1.1.0-SNAPSHOT https://github.com/openmrs/openmrs-module-billing/blob/main/pom.xml#L8

Reply:
We didn't *set* that to false, it must have been a default value and we didn't catch it. We will fix it.

Reply:
Could there be a reason why you are not using the latest snapshot version `1.2.0-SNAPSHOT` of this module to take advantage of the latest changes? Just like you have done for the stockmanagement module.